### ISSUE TITLE
PS/2 socket footprint mirrored in r1

### ISSUE THREAD

**Issue by rpavlik** [rpavlik/PS2-Breakout-Board#7]
This would be my big "oopsie" for the R1 board - I apparently got the pinout of the port mirrored when making the footprint. This means a few things, if you connect a port and keyboard without board rework:

- Channel 1 (data/clk) and Channel 2 (data2/clk2) are swapped - no big deal except...
- GND and VCC are swapped (so now you're pulling down to gnd and you've got vcc on your ground plane, as well as your shield, etc... - basically a dead short once you plug a keyboard in...)

### The hard way
- On the board front, cut a trace (to what I thought was VCC) and the thermal connections to the ground plane (to what I thought was GND) 
![my_photo-22-annotated](https://user-images.githubusercontent.com/61129/28678561-50f49e82-72b6-11e7-8a7c-86f5671c7838.jpg)

- After mounting the port, solder small wires on the back of the board to replace those traces. 
![my_photo-25-annotated](https://user-images.githubusercontent.com/61129/28678571-568f0b16-72b6-11e7-90be-02a720df084f.jpg)

- Use the pullup pads and pinouts labeled for ch2 instead of ch1, since they're swapped.

### The easy way
Of course, realized only after I had already done the above. If your mounting requirements allow (and frankly, you could realistically change all board component mounting since it's pretty simple and all through-hole...)

*put the PS/2 port on from the "back side" of the board* - this will mirror the pins back, and everything will work the way it's supposed to.  (facepalm)
